src/hooks/useRadioButton.ts

```diff
@@ -0,0 +1,13 @@
+import { useState } from "react";
+
+import { RadioContextState } from "@/store/context/RadioContext";
+
+type UseRadioButtonReturn = [RadioContextState["isChecked"], RadioContextState["onChange"], string | undefined];
+
+export const useRadioButton = (): UseRadioButtonReturn => {
+  const [selectedValue, setSelectedValue] = useState<string>();
+  const isChecked = (value: string) => selectedValue === value;
+  const onChange = (value: string) => setSelectedValue(value);
+
+  return [isChecked, onChange, selectedValue];
```

Comment:
```suggestion

export const useRadioButton = () => {
  const [selectedValue, setSelectedValue] = useState<string>();
  const isChecked = (value: string) => selectedValue === value;
  const onChange = (value: string) => setSelectedValue(value);

  return [isChecked, onChange, selectedValue] as const;
```

저는 custom hook에서 배열로 반환 시 return type을 설정해줄 때 as const로 명시해주는게 깔끔하더라구요~!!

별개로 object return 이 아닌 array return을 채택하신 이유도 궁금해요~!!

Reply:
아!! 맞네요 as const로 수정하겠습니다! 감사합니다 🙇‍♀️

+) [Staging.tsx](https://github.com/depromeet/layer/pull/27/files#diff-17b93f0c3abe049bdbe4a710b3b2ef85d2094301ed5c40f0b13d40ddbb01b12f) 파일에 예시 작성하면서 라디오버튼과 체크박스를 같이 사용할 때 `useRadioButton`, `useCheckBox` 훅이 같은 이름(isChecked)을 반환하다보니 충돌이 나더라구요! 저희 프로젝트에서는 아직까지 그럴 일은 없겠지만, 그래도 이름을 사용하는 쪽에서 바꿔서 사용하면 좋겠다는 생각으로 이렇게 작성했습니다!

Reply:
자세한 설명 감사합니다👍👍